content/user/ui/overview/Basic-quick-start-guide.md

```diff
@@ -0,0 +1,111 @@
+<!--
+title: "Quick Start Guide"
+description: "Overview of Contrast UI for new users"
+tags: "TeamServer user applications vulnerabilities"
+-->
+
+Welcome to Contrast! This is your guide to everything you need to know to get started in the interface. Start with onboarding your applications and then move on to environments, libraries and vulnerabilities – you’ll be up and running by the time you’re done. As always, if you have any more questions, let the Contrast Support team know by emailing <support@contrastsecurity.com>. 
```

Comment:
Can we revise "...you'll be up and running by the time you're done." ? It's really the "by the time you're done" part that bothers me. Maybe just "up and running before you know it." ?